django/tasks/utils.py

```diff
@@ -0,0 +1,84 @@
+import inspect
+import json
+import time
+from collections import deque
+from functools import wraps
+from traceback import format_exception
+
+from django.utils.module_loading import import_string
+
+
+def is_global_function(func):
```

Comment:
I've heard some comments that "global" here could be confusing. Perhaps `is_module_level_function` is a better name?